lib/srv/desktop/windows_server.go

```diff
@@ -865,6 +866,10 @@ func (s *WindowsService) connectRDP(ctx context.Context, log logrus.FieldLogger,
 		Tracker:           rdpc,
 		TeleportUser:      identity.Username,
 		ServerID:          s.cfg.Heartbeat.HostUUID,
+		MessageWriter: &monitorErrorSender{
```

Comment:
This doesn't look properly formatted.

Reply:
I agree, but that's what it gives me 🤷‍♂️

Reply:
huh. same here 🤷‍♂️